### PR TITLE
128-documentation

### LINKED ISSUE #128: Confirm/Update Documentation and Original Requirements
### Feature Description

See [Definition of Done](https://github.com/nashville-software-school/bangazon-inc/blob/master/EMPLOYEE_HANDBOOK.md#definition-of-done)
### Recommended Actions

> Refer to the [project requirements](https://github.com/TeamCharles/bangazon-inc/blob/master/projects/INITIAL_BANGAZON_SITE.md#requirements) before filling out this section.
1. Check every Class and Method for proper documentation
   - If in doubt, document
2. Check final product against original requirements and any updated requirements
3. Check final product against all User Story Issues

### PR DESCRIPTION
**Pull request by MattKraatz**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Update documentation on all controllers

## Resolves Issue Number
Resolves #128 

## Deploy Notes

```sh
git pull --prune
git checkout "128-documentation"
dotnet run
```

## Steps to Test or Reproduce

1. Read the code bro

## Impacted Files in Application
List general components of the application that this PR will affect:

* All Controller Classes

### CODE REVIEW

File: Controllers/ProductSubTypesController.cs [hunk 1]
```diff
@@ -16,10 +16,12 @@ namespace BangazonWeb.Controllers
...
      * METHODS:
-     *   Task<IActionResult> Index() - Shows all product types and counts
-     *   Task<IActionResult> list() - Shows all products that match a specified ProductTypeId
-     *   CalculateTypeQuantities(ProductType) - Queries the Product table to return a new ProductType object...
-     *        ...new ProductType object contains a value for Quantity, based on number of Products with that Type
+     *   Task<IActionResult> List(int id) - Returns a view for all ProductSubTypes with a given ProductTypeId.
+     *          - int id: ProductTypeId for the ProductSubTypes being requested to view. 
+     *   Task<IActionResult> Products(int id) - Returns a view for all products with a given ProductSubTypeId.
+     *          - int id: ProductSubTypeId for the Products being requested to view. 
+     *   CalculateTypeQuantities(ProductSubType productSubType) - Queries the Product table to count the number of Products in a given ProductSubType. Updates the ProductSubType.Quantity property.
```
Comment (matthamil) [inline Controllers/ProductSubTypesController.cs]: `CalculateTypeQuantities(ProductSubType productSubType)` doesn't explicitly say it's return type of `void`.

In `UserController.cs`, the return type of `void` is added to one of the methods but it's not here.

**Review by matthamil — CHANGES_REQUESTED** [review]
Minor fix

**Comment by garrett-vangilder** [comment]
LGTM

File: Controllers/ProductSubTypesController.cs (hunk 1 above)
Comment (MattKraatz) [reply Controllers/ProductSubTypesController.cs]: 👍

**Review by matthamil — APPROVED** [review]

**Pull request merged**